mods/voidmulti.sc2mod/base.sc2data/gamedata/abildata.xml

```diff
@@ -202,6 +224,11 @@
         <CmdButtonArray index="Execute">
             <Flags index="ShowInGlossary" value="0"/>
         </CmdButtonArray>
+        <InfoArray index="0">
+            <SectionArray index="Stats">
+                <DurationArray index="Duration" value="0"/>
```

Comment:
**undocumented**: Ravager's Unburrow stats transition duration is now changed from 0.4443 to 0

Does this cause unburrowing units to become "visible" immediately instead of after ~300ms? Does this change anything?